.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 npm-debug.log
 dev/
 coverage/
+.idea/
```

Comment:
You can add this to [global gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) in your machine. If this repository had to list all editor/OS specific files to ignore, it would be a very long list.